src/elements/content-explorer/ContentExplorer.js

```diff
@@ -391,14 +391,10 @@ class ContentExplorer extends Component<Props, State> {
 
         // New folder state
         const newState = {
-            selected: undefined,
             currentCollection: collection,
             rootName: id === rootFolderId ? name : '',
         };
 
-        // Unselect any rows that were selected
-        this.unselect();
```

Comment:
Why is this no longer needed?

Reply:
In some cases, this function is called after closing a modal.  In this scenario, the item that was selected before opening the modal should be selected after closing the modal.

I call this.updateCollection() instead of this.unselect() now.  this.updateCollection() will keep the current item selected if it is in the folder, otherwise it will make everything unselected.